content/aips/35A56F18FEFC-AIP-REDEEM-AFEI-FOR-DAI.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Ethereum v2 Reserve Factor - aFEI Holding Update
+status: Proposed
+author: Llama, Matthew Graham & defijesus
+shortDescription: Redeem aFEI for DAI
+discussions: https://governance.aave.com/t/arc-ethereum-v2-reserve-factor-afei-holding-update/9401
+created: 28-09-2022
```

Comment:
The date format seems to be wrong. Check the other AIP files: It should be like `2022-09-28`. This means that after this change, you'll need to recompute the hash for the filename as well since it depends on this date.